File: app/src/main/java/com/wealoha/social/beans/user/PromotionGetData.java
package com.wealoha.social.beans.user;

import java.io.Serializable;

import com.wealoha.social.beans.ResultData;

/**
 * @author javamonk
 * @createTime 14-10-16 PM12:02
 */
public class PromotionGetData extends ResultData implements Serializable {

	private static final long serialVersionUID = -6138532041290176835L;
	public static final String TAG = PromotionGetData.class.getSimpleName();
	public String promotionCode;
	public int quotaReset;
	public int inviteNewUserCount;
	public boolean alohaGetLocked;
	public int alohaGetUnlockInviteNeed;
	public int alohaGetUnlockMoreInviteNeed;
	public int alohaGetUnlockMax;
	public int quotaPerPerson;

}

File: app/src/main/java/com/wealoha/social/beans/user/PromotionGetData.kt
package com.wealoha.social.beans.user

import com.wealoha.social.beans.ResultData
import java.io.Serializable

/**
 * @author javamonk
 * @createTime 14-10-16 PM12:02
 */
class PromotionGetData : ResultData(), Serializable {
    @kotlin.jvm.JvmField
    var promotionCode: String? = null
    @kotlin.jvm.JvmField
    var quotaReset = 0
    var inviteNewUserCount = 0
    @kotlin.jvm.JvmField
    var alohaGetLocked = false
    var alohaGetUnlockInviteNeed = 0
    var alohaGetUnlockMoreInviteNeed = 0
    var alohaGetUnlockMax = 0
    @kotlin.jvm.JvmField
    var quotaPerPerson = 0

    companion object {
        private const val serialVersionUID = -6138532041290176835L
        @kotlin.jvm.JvmField
        val TAG = PromotionGetData::class.java.simpleName
    }
}